core/decoders/tight.js

```diff
@@ -78,8 +78,8 @@ export default class TightDecoder {
             return false;
         }
 
-        const rQi = sock.get_rQi();
-        const rQ = sock.rQwhole();
+        const rQi = sock.rQi;
+        const rQ = sock.rQ;
```

Comment:
Is this safe? `rQwhole()` gave you a new object, rather than a reference to an existing one.

Reply:
`rQwhole` made a clone of the object.
That's safer but let's performant.

This function is the only place where `rQwhole` was used and it only reads from the variable but never modifies it.